orchagent/portsorch.cpp

```diff
@@ -3971,8 +3971,13 @@ void PortsOrch::flushFDBEntries(sai_object_id_t bridge_port_id)
     vector<sai_attribute_t> attrs;
     sai_status_t rv;
 
-    SWSS_LOG_INFO("Flushing the port with bridge port id: %" PRIx64, bridge_port_id);
+    if (SAI_NULL_OBJECT_ID == bridge_port_id)
+    {
+        SWSS_LOG_WARN("null bridge port ID");
```

Comment:
Can you make this message more clear that it won't flush any fdb entries?

Reply:
If the bridge_port_oid is SAI_NULL_OBJECT_ID, all L2 entries associated with all ports will be removed.

Reply:
I understand the problem, I was meant for the message in the code. I saw you changed.